Source/SDK/ClientBuilder.cs

```diff
@@ -364,5 +448,27 @@ async Task EventHorizonRetryPolicy(Subscription subscription, ILogger logger, Fu
                 await Task.Delay(timeout).ConfigureAwait(false);
             }
         }
+
+        ClientBuilder WithAllScannedAssemblies(Action<Assembly> doWithAssembly, Func<Assembly, bool> assemblyFilter = default, bool includeExeFiles = false)
+        {
+            foreach (var assembly in GetAssemblies(assemblyFilter, includeExeFiles))
+            {
+                doWithAssembly(assembly);
+            }
+
+            return this;
+        }
+
+        Task RegisterEventTypes(MethodCaller methodCaller, ExecutionContext executionContext, EventTypes eventTypes)
+            => new Events.Internal.EventTypes(methodCaller, executionContext, _loggerFactory.CreateLogger<Events.Internal.EventTypes>()).Register(eventTypes, _cancellation);
```

Comment:
Can we make a client for this thing as well like the aggregate roots thing - I think that looks nicer (and then they are the same) :)